charts/latest/csi-driver-smb/values.yaml

```diff
@@ -138,6 +138,10 @@ windows:
       requests:
         cpu: 10m
         memory: 40Mi
+  containerSecurityContext:
```

Comment:
this PR is different from https://github.com/kubernetes-csi/csi-driver-nfs/pull/422

Reply:
removed